Boards/arduino_mega.board.json

```diff
@@ -5,7 +5,6 @@
     "BaudRates": [ "115200" ],
     "Programmer": "wiring",
     "FirmwareBaseName": "mobiflight_mega",
-    "ResetFirmwareFile": "reset.arduino_mega_1_0_2.hex",
```

Comment:
This property got moved to the `Info` object so it can be shared by both COM and USB-based boards.

Reply:
I moved `FirmwareBaseName` and `FirmwareExtension` into the `Info` object since it was already there and meant less disruption to the schema. There was already firmware-related stuff there too, like the latest firmware version.